`wasm-rt-impl-tableops.inc` missing from packages

`wasm-rt-impl-tableops.inc` has been [split off from `wasm2c/wasm-rt-impl.c`](https://github.com/WebAssembly/wabt/commit/49950cbcb8e3729cd056f1f76ee4000dbcbe309f), but this file is not installed in the installation dir. This means that `wasm-rt-impl.c` is no longer complete in installed packages.

E.g., on Homebrew:

```
cc -c -DVERSION='"0.8.0"' -Wall -Wno-unused-function -Wno-unused-value -I/opt/homebrew/include -O2 -o wasm-rt-impl.o /opt/homebrew/share/wabt/wasm2c/wasm-rt-impl.c
/opt/homebrew/share/wabt/wasm2c/wasm-rt-impl.c:262:10: fatal error: 'wasm-rt-impl-tableops.inc' file not found
#include "wasm-rt-impl-tableops.inc"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Same goes for `wasm-rt-mem-impl-helper.inc`

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -423,7 +423,7 @@ if (WABT_INSTALL_RULES)
 endif ()
 
 if (HAVE_SETJMP_H)
-  set(WASM_RT_FILES "wasm2c/wasm-rt-impl.h" "wasm2c/wasm-rt-impl.c" "wasm2c/wasm-rt-exceptions-impl.c" "wasm2c/wasm-rt-mem-impl.c")
+  set(WASM_RT_FILES "wasm2c/wasm-rt-impl.h" "wasm2c/wasm-rt-impl.c" "wasm2c/wasm-rt-exceptions-impl.c" "wasm2c/wasm-rt-mem-impl.c" "wasm2c/wasm-rt-impl-tableops.inc" "wasm2c/wasm-rt-mem-impl-helper.inc")
 
   add_library(wasm-rt-impl STATIC ${WASM_RT_FILES})
   target_link_libraries(wasm-rt-impl ${CMAKE_THREAD_LIBS_INIT})

